homeassistant/components/weather/__init__.py

```diff
@@ -97,6 +98,11 @@ def attribution(self):
         """Return the attribution."""
         return None
 
+    @property
+    def time_zone(self):
```

Comment:
Please first open an issue in our architecture repo and present what, why and how about this change.

https://developers.home-assistant.io/docs/en/entity_index.html#changing-the-entity-model

Reply:
Done on https://github.com/home-assistant/architecture/issues/221, thank you for the hint.